src/components/Greeting.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchMessage } from '../redux/features/greetingSlice';
+
+function Greeting() {
+  const { message } = useSelector((state) => state.greeting);
+  const dispatch = useDispatch();
+
+  React.useEffect(() => {
+    dispatch(fetchMessage());
+  }, [dispatch]);
+
+  return (
+    <div>
+      <h1>{message.greeting}</h1>
```

Comment:
* I get CORS errors in my console when I try to run your app, Kindly [Install the CORS gem and follow the configurations to fix this error](https://github.com/cyu/rack-cors) 
![image](https://github.com/yosaddis/hello-rails-back-end/assets/52497246/41b4d1b5-eef8-4207-bc4a-a5d6bf57a2c8)